test/doc-views/layout/spec.handlebars

```diff
@@ -135,7 +135,9 @@
             </p>
         {{/if}}
         {{#if copyrightHtmlContent.show}}
-            {{{copyrightHtmlContent.html}}}
+            <p class="copyright">
```

Comment:
It's starting to be confusing to have so many copyright parameters. I think it might be better to have something like that:
```
<p class="copyright">
{{#if copyright.default}}
    <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">{{copyright.startText}}</a> © {{nowYear}} <a href="{{copyright.W3CLink}}"> World Wide Web Consortium</a>. <abbr title="World Wide Web Consortium">W3C</abbr><sup>®</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a href="https://www.w3.org/Consortium/Legal/copyright-software">permissive document license</a> rules apply.
{{else}}
    {{{copyright.html}}}
{{/if}}
</p>
```